wireguard.sh

```diff
@@ -58,8 +54,10 @@ latest_release_for() {
 	# > The created_at attribute is the date of the commit used for the
 	# > release, and not the date when the release was drafted or published.
 	curl -sSL https://api.github.com/repos/${WIREGUARD_REPO}/releases \
-		| jq -r --arg version "wireguard-${board}" \
-			'sort_by(.created_at) | reverse | .[0].assets | map(select(.name | contains($version))) | .[0] | {name: .name, url: .browser_download_url}'
+		| jq -r --arg version "${board}" \
+			'sort_by(.created_at) | reverse | .[0] | .tag_name as $tag_name \
```

Comment:
Can you confirm these `\` inside the qutes work as intended on your device? I'm seeing this on my USG:

```
error: syntax error, unexpected INVALID_CHARACTER, expecting '|'
sort_by(.created_at) | reverse | .[0] | .tag_name as $tag_name \
                                                               ^
1 compile error
```

The USG has bash 4.2.37, and bash does support multi-line strings (i.e. `\` is interpreted literally here), but I had still expected this to work 😅.

Reply:
You're absolutely right! The version I tested does not use the linebreaks, I must have added them afterwards. Now tested without backslashes, then it works. Fix imminent.